inflection/src/inflection/tokenizer/TokenizerFactory.cpp

```diff
@@ -146,7 +146,7 @@ Tokenizer* TokenizerFactory::createTokenizerObject(const util::ULocale& locale,
                 }
                 auto tokenizer = new Tokenizer(TOKENIZER_CONSTRUCTORS[mid].construct(locale, systemConfig));
                 // Initialize the data loading and caches so that it doesn't possibly fail later.
-                delete npc(tokenizer)->createTokenChain(u"initialization");
+                ::std::unique_ptr<TokenChain> unused(npc(tokenizer)->createTokenChain(u"initialization"));
```

Comment:
The original was actually correct. There is no intermediate line between getting the object and deleting it. I recommend leaving this one alone.